restock-shop.lic

```diff
@@ -12,7 +12,9 @@ class ShopRestock
   include DRCT
 
   def initialize
-    ensure_copper_on_hand(40_000, get_settings.hometown)
+    settings = get_settings
+    hometown = settings.hometown
```

Comment:
Can `hometown` be deleted? I don't see it used anywhere

Reply:
Yes.